lib/parser.js

```diff
@@ -134,23 +134,24 @@ function parseFromUrl(url, fetchOptions = {}, options) {
 async function dereference(parsedJSON, initialFormat, asyncapiYAMLorJSON, options) {
   try {
     return await refParser.dereference(options.path, parsedJSON, {
+      continueOnError: true,
```

Comment:
new flag from [release v9](https://github.com/APIDevTools/json-schema-ref-parser/blob/master/CHANGELOG.md#v900-2020-04-21) that allows you to get all errors rather than just the first one